src/sctools/metrics/aggregator.py

```diff
@@ -246,10 +246,8 @@ def parse_molecule(
             all the tags that define this molecule. one of {[CB, GE, UB], [GE, CB, UB]}
         records : Iterable[pysam.AlignedSegment]
             the sam records associated with the molecule
-
         """
         for record in records:
-
             # todo think about how I could use the duplicate tag to reduce computation; duplicates
```

Comment:
Remove this.

Reply:
still there

Reply:
@kishorikonwar these comments have not been addressed as you said they were. This is not ready for re-review. Please address the comments that you said you have addressed before requesting another review.